OloEngine/tests/SlugFontRenderingTest.cpp

```
@@ -0,0 +1,232 @@
+                hasMove = true;
+            if (vertices[i].type == STBTT_vcurve || vertices[i].type == STBTT_vline)
+                hasCurve = true;
+        }
+        EXPECT_TRUE(hasMove) << "Glyph " << cp << " should have MOVE vertices";
+        EXPECT_TRUE(hasCurve) << "Glyph " << cp << " should have CURVE/LINE vertices";
+
+        stbtt_FreeShape(&m_FontInfo, vertices);
+    }
+}
+
+TEST_F(SlugFontProcessorTest, SpaceGlyphHasNoCurves)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)